climada/hazard/test/test_tc_cc.py

```diff
@@ -26,40 +26,51 @@
-        self.assertEqual(criterion[-10]['basin'], "SP")
-        self.assertEqual(criterion[-10]['variable'], "frequency")
-        self.assertEqual(criterion[-10]['change'], 1 - 0.583)
-
-    def test_scale_pass(self):
-        """Test calc_scale_knutson function."""
-        self.assertAlmostEqual(tc_cc.calc_scale_knutson(ref_year=2050, rcp_scenario=45),
-                               0.759630751756698)
-        self.assertAlmostEqual(tc_cc.calc_scale_knutson(ref_year=2070, rcp_scenario=45),
-                               0.958978483788876)
-        self.assertAlmostEqual(tc_cc.calc_scale_knutson(ref_year=2060, rcp_scenario=60),
-                               0.825572149523299)
-        self.assertAlmostEqual(tc_cc.calc_scale_knutson(ref_year=2080, rcp_scenario=60),
-                               1.309882943406079)
-        self.assertAlmostEqual(tc_cc.calc_scale_knutson(ref_year=2090, rcp_scenario=85),
-                               2.635069196605717)
-        self.assertAlmostEqual(tc_cc.calc_scale_knutson(ref_year=2100, rcp_scenario=85),
-                               2.940055236533517)
-        self.assertAlmostEqual(tc_cc.calc_scale_knutson(ref_year=2066, rcp_scenario=26),
-                               0.341930203294547)
-        self.assertAlmostEqual(tc_cc.calc_scale_knutson(ref_year=2078, rcp_scenario=26),
-                               0.312383928930456)
+        criterion = tc_cc.get_knutson_scaling_factor(
+                    percentile='5/10',
+                    baseline=(1950, 2018))
+        self.assertEqual(criterion.shape, (21, 4))
+
+        self.assertEqual(criterion.columns[0], '2.6')
+        self.assertEqual(criterion.columns[1], '4.5')
+        self.assertEqual(criterion.columns[2], '6.0')
+        self.assertEqual(criterion.columns[3], '8.5')
+
+        self.assertAlmostEqual(criterion.loc[2030, '2.6'], -16.13547, 4)
+        self.assertAlmostEqual(criterion.loc[2050, '4.5'], -25.19448, 4)
+        self.assertAlmostEqual(criterion.loc[2070, '6.0'], -31.06633, 4)
+        self.assertAlmostEqual(criterion.loc[2100, '8.5'], -58.98637, 4)
+
+    def test_get_gmst_pass(self):
+        """Test get_gmst_info function."""
+        gmst_info = tc_cc.get_gmst_info()
+
+        self.assertAlmostEqual(gmst_info['gmst_start_year'], 1880)
+        self.assertAlmostEqual(gmst_info['gmst_end_year'], 2100)
+        self.assertAlmostEqual(len(gmst_info['rcps']), 4)
+
+        self.assertAlmostEqual(gmst_info['gmst_data'].shape,
+                               (len(gmst_info['rcps']),
+                                gmst_info['gmst_end_year']-gmst_info['gmst_start_year']+1))
+        self.assertAlmostEqual(gmst_info['gmst_data'][0,0], -0.16)
+        self.assertAlmostEqual(gmst_info['gmst_data'][0,-1], 1.27641, 4)
+        self.assertAlmostEqual(gmst_info['gmst_data'][-1,0], -0.16)
+        self.assertAlmostEqual(gmst_info['gmst_data'][-1,-1], 4.477764, 4)
+
+    def test_get_knutson_data_pass(self):
```

Comment:
Could you please write this as a simple unit test (it must be plainly obvious when the answer is correct or not). Using more complex / real data is better for an integration test.